docs/ref/contrib/postgres/indexes.txt

```diff
@@ -7,6 +7,26 @@ PostgreSQL specific model indexes
 The following are PostgreSQL specific :doc:`indexes </ref/models/indexes>`
 available from the ``django.contrib.postgres.indexes`` module.
 
+``BloomIndex``
+==============
+
+.. class:: BloomIndex(length=None, columns=None, **options)
+
+    .. versionadded:: 3.1
+
+    Creates a `bloom index
+    <https://www.postgresql.org/docs/current/bloom.html>`_.
+
+    To use this index access method you need to activate the `bloom extension
+    <https://www.postgresql.org/docs/current/bloom.html>`_ on PostgreSQL. You
```

Comment:
Please move `https://www.postgresql.org/docs/current/bloom.html` below and use it in both places:
```
    ... bloom: https://www.postgresql.org/docs/current/bloom.html
```